File: step5/2577.c
#include <stdio.h>

int main(void)
{
    int A, B, C, num, cnt[10] = { 0 };
    scanf("%d", &A);
    scanf("%d", &B);
    scanf("%d", &C);
    num = A * B * C;
    while (num / 10 != 0) {
        cnt[num % 10]++;
        num /= 10;
    }
    cnt[num]++;
    for (int i = 0; i < 10; i++) {
        printf("%d\n", cnt[i]);
    }
    return 0;
}

File: step5/2577.py
A = int(input())
B = int(input())
C = int(input())
num = A * B * C
cnt = [0 for i in range(10)]
while num // 10 != 0:
    cnt[int(num % 10)] += 1
    num //= 10
cnt[num] += 1
for i in range(10):
    print(cnt[i])
